### ISSUE TITLE
where to max the PriorDiscriminator?

### ISSUE THREAD

**Issue by bojone** [rcalland/deep-INFOMAX#1]
the prior term of deep infomax is a minimax game, like GAN. But I can not find any code about it in train.py. Do I have a wrong understand or your implement has some problem?

**Comment by mm04926412** [comment]
Equally confused, this error has propogated to some "child" projects if it is indeed an error.

**Comment by rcalland** [comment]
you're right, I made a mistake and there should be an update step for the prior discriminator
